iaso/api/mobile/entity_type.py

```diff
@@ -0,0 +1,99 @@
+from django.core.exceptions import ValidationError
+from django.db.models import Prefetch
+from django_filters.rest_framework import DjangoFilterBackend  # type: ignore
+from rest_framework import filters, status
+from rest_framework import serializers
+from rest_framework.decorators import action
+from rest_framework.pagination import PageNumberPagination
+from rest_framework.response import Response
+
+from iaso.api.common import (
+    ModelViewSet,
+    TimestampField,
+)
+from iaso.api.mobile.entity import MobileEntitySerializer
+from iaso.models import Entity, Instance, EntityType
+from iaso.utils.jsonlogic import jsonlogic_to_q
+
+
+class LargeResultsSetPagination(PageNumberPagination):
+    page_size = 1000
+    page_size_query_param = "page_size"
+    max_page_size = 1000
```

Comment:
I think it's probably a good idea to reuse the same class as the one in `entity.py` rather than copy/pasting it here